confidential-data-hub/kms/src/plugins/aliyun/client/client_key_client/config.rs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2023 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+//! Configs to access aliyun KMS
+
+use anyhow::*;
+
+#[derive(Clone, Debug)]
+pub(crate) struct ConfigClientKey {
+    pub kms_instance_id: String,
+    pub endpoint: String,
+}
+
+// implement ConfigClientKey related function
+impl ConfigClientKey {
+    pub(crate) fn new(kms_instance_id: &str, endpoint: &str) -> Result<Self> {
```

Comment:
If we do not return any `Error`, usually we do not use `Result<Self>` but directly `Self`

Reply:
`Result<_>` removed.